packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-accessors--to-es2015/context-name/output.js

```diff
@@ -1,4 +1,5 @@
-var _initStatic, _init_a, _init_a2, _get_a, _set_a, _init_computedKey, _init_computedKey2, _init_computedKey3, _init_computedKey4, _init_computedKey5, _init_computedKey6, _computedKey, _init_computedKey7, _Foo;
+let _computedKey;
+var _initStatic, _init_a, _init_a2, _get_a, _set_a, _init_computedKey, _init_computedKey2, _init_computedKey3, _init_computedKey4, _init_computedKey5, _init_computedKey6, _init_computedKey7, _Foo;
```

Comment:
Can we revert this change?

Reply:
This change is intentional as it will fix an integration issue with the class properties transform. For context see https://github.com/babel/babel/pull/10029/files#r287785963 

In this [REPL example](https://babel.dev/repl#?browsers=chrome%2032&build=&builtIns=false&corejs=3.21&spec=false&loose=true&code_lz=GYVwdgxgLglg9mABGOcAOAKAlIg3gXwCgIEBnKRCAGwENTSBTUxAXkQG0BdAbkODgBOiDFQYUYrRAAZuiCQB5EAVlkBqVTBy5CiSrXpMAdGhCkAFhh269dZgEE8V64gACKdE-vsYnSQCIAWwBPRDQBdAYBKCC_XmciXSxeIgA3GiEBJhAqCjYuXn4hDBIwchsAL3LEOGAbA1ItK0zSbKhjUwsAeQAjACsGaEMAczFOgHcwAAVwtEjogDkaAKYMMAYxivLsLHYpTiTCIhLSOFFDKjghjGbWw164GDBtoA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=&prettier=false&targets=&version=7.24.0&externalPlugins=%40babel%2Fplugin-proposal-decorators%407.23.3%2C%40babel%2Fplugin-transform-class-properties%407.23.3%2C%40babel%2Fplugin-transform-class-static-block%407.23.4&assumptions=%7B%7D)

```js
function noop() {}
const classes = [];
for (let i = 0; i < 5; ++i) {
  classes.push(
    class A {
      @noop
      [i] = "my property";
    }
  );
}
var result = [];
for (const clazz of classes) {
  result.push(Object.getOwnPropertyNames(new clazz())[0]);
}
console.log(result.join())
```
when we apply the decorators, static block and class properties plugin, it should print `0,1,2,3,4` but currently it prints `4,4,4,4,4`.

If the output size is the concern, I can open a new PR to register all variable as the `let` binding.

Reply:
Yes please add a test!

Regarding to output size, we should first aim at correctness and _then_ optimization.

Reply:
Thanks! I didn't notice this was a bug fix, I thought the increased size didn't matter.

Reply:
2cd5bcc2007b8e9fea907bc96eb5606334f038a3